src/test/forward_index_test.cpp

```diff
@@ -128,9 +128,25 @@ int forward_index_tests()
     num_failed += testing::run_test("forward-index-read-file-corpus", [&]()
     {
         ceeaus_forward_test();
-        system("rm -rf ceeaus-* test-config.toml");
     });
 
+    num_failed += testing::run_test("forward-index-build-uninvert", [&]()
+    {
+        system("rm -rf ceeaus-*");
+
+        // hack to inject "uninvert = true" at the top of the config file
+        auto cfg_contents = filesystem::file_text("test-config.toml");
+        cfg_contents = "uninvert = true\n" + cfg_contents;
+        filesystem::delete_file("test-config.toml");
```

Comment:
Also, are the unit tests actually testing double values? I think the datasets we use are ceeaus and breast-cancer; breast-cancer has "2.0000" or "4.0000" as class labels, but no values like "2.3546". Or does it not even make sense to test this until we have regression implemented?